src/runmode-netmap.c

```diff
@@ -53,7 +53,7 @@
 #include "util-ioctl.h"
 #include "util-byte.h"
 
-#ifdef HAVE_NETMAP
+#if HAVE_NETMAP && USE_NEW_NETMAP_API
```

Comment:
This change will produce a later compilation error as both the legacy netmap API code and the new v14 multiple host stack rings code needs the <netmap_user.h> header included. Both also need the NETMAP_WITH_LIBS conditional defined.